encoding/descaling.md

```diff
@@ -1,477 +1,365 @@
-It's instead recommended to resize
-as you would normally,
-or to not mess with scaling at all.
+Among the many tools available in an encoder's arsenal,
+the ability to descale might arguably be one of the best.
+But what does descaling actually mean?
+In what way does it differ from normal downscaling?
+How do you actually descale properly?
+This guide will explain the groundwork for you
+to start descaling effectively.
+
+
+## Descaling vs. Downscaling
+
+What the difference between downscaling and descaling is
+is a question that gets asked fairly frequently.
+Simply put,
+downscaling is the act of resizing an image
+to a smaller resolution than the original image,
+while with descaling you are actively reversing the math
+applied to an upscale,
+reconstructing the original image before upscaling.
+
+Descaling is generally the preferred method
+of handling upscaled content whenever possible.
+When performing a simple downscale,
+information will be lost in the process.
+A proper descale will, on the other hand,
+(theoretically) preserve all the detail
+that was actually there in the original image.
+This offers a multitude of advantages,
+like for example when dealing with artifacting
+that was a result of a poor upscale.
+You could instead undo the scaling
+and use a better scaler to upscale it instead,
+or keep it at its native resolution and
+have less artifacting to worry about.
+
+For a more thorough explanation on the intrinsics of descaling,
+please refer to the [vapoursynth-descale Github page][vapoursynth-descale].
+
+That said, if you can't perform a good descale,
+you will usually end up with haloing/ringing artifacting,
+as illustrated below:
+
+![Lineart messed up due to poor descaling](images/descale/descale_ubw_00_bad.png)
+
+<div class="warning box"><p>
+Only descale if you can be sure,
+beyond reasonable doubt,
+that you have figured out the correct resolution and kernel.
+</p></div>
+
+[vapoursynth-descale]: https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale
+
+
+# Discerning the native resolution
+
+The first step to descaling is to figure out
+what the native resolution of a frame is.
+One way to do this is by descaling to all potential resolutions,
+upscaling the descaled image with the same kernel,
+and comparing the two.
+The artifacts created by descaling to a wrong resolution
+are generally more obvious than
+the artifacts created by using the wrong kernel.
+Luckily for us,
+there's already a tool out there
+you can use for this.
+
+[getnative.py][getnative] is a common tool
+used for figuring out the native resolution of an image.
+It will return a graph showcasing all the resolutions
+it tried to descale to,
+as well as a plaintext file
+with all the relative errors listed.
+Here's an example:
+
+![](images/descale/descale_owari_01_frame.png)
+
+![Results from "getnative.py -k bicubic -b 0 -c 1/2"](images/descale/descale_owari_01_graph.png)
+
+What you want to look out for are very clear spikes.
```

Comment:
```suggestion
What you want to look out for are very clear downward spikes.
```
I've seen a few awful graphs in the #bot channel that have spikes somehow going up, and we'd want to avoid these afaict.